TrailBlaze/Objects/Interception.h

```diff
@@ -0,0 +1,22 @@
+//
+//  Interception.h
+//  TrailBlaze
+//
+//  Created by Adam Issah on 7/26/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "MapKit/Mapkit.h"
+@import Parse;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Interception : PFObject<PFSubclassing>
```

Comment:
is this meant to represent the intercepting route? the name isn't very clear imo

Reply:
It contains the route as well as the rendezvous point, the start time and the intercepting user's id. Since all these have to do with the entire interception and not just the route I made the name as such.